docs/cargo_features.md

```diff
@@ -51,6 +51,7 @@ The default feature set enables most of the expected features of a game engine,
 |dynamic_linking|Force dynamic linking, which improves iterative compile times|
 |exr|EXR image format support|
 |flac|FLAC audio format support|
+|glam_assert| Enable assertions to check the validity of parameters passed to glam|
```

Comment:
```suggestion
|glam_assert|Enable assertions to check the validity of parameters passed to glam|
```
This line can be generated for you if you run `cargo run -p build-templated-pages -- update features`